Example/PrebidInternalTestApp/src/main/java/org/prebid/mobile/renderingtestapp/plugplay/bidding/gam/original/GamOriginalInstreamNewApiFragment.kt

```diff
@@ -101,7 +101,7 @@ class GamOriginalInstreamNewApiFragment : AdFragment() {
 
     override fun onDestroyView() {
         super.onDestroyView()
-        adUnit?.stopAutoRefresh()
```

Comment:
This change looks as potential memory leak. What if the publisher utilizes a new counting methodology and will not call `destroy()`?

Reply:
If a user uses this burl tracker, it's better to use destroy(). Otherwise the monitor will traverse view hierarchy as long as the ad view container exists.